compiler/testData/diagnostics/testsWithJvmBackend/inlineCycle.kt

```diff
@@ -1,11 +1,11 @@
 // !RENDER_DIAGNOSTICS_FULL_TEXT
-
+// IGNORE_BACKEND: JVM_OLD
```

Comment:
Looks like this was the only test on `INLINE_CALL_CYCLE` on the old JVM backend, so could you please keep a copy with target JVM_OLD?

Reply:
Done. (BTW, why is it JVM_OLD and not JVM?)

Reply:
What I meant was to use `TARGET_BACKEND: JVM_OLD` instead of `IGNORE_BACKEND: JVM_IR`, so that we wouldn't need to come back to this test and wonder why it's still failing for JVM_IR, since the former is a bit more explicit :). The other tests in `diagnostics/testsWithJvmBackend` are handled in this way exactly for this reason.

AFAIK `JVM` and `JVM_OLD` are handled differently only in `TARGET_BACKEND`. `IGNORE_BACKEND: JVM` and `IGNORE_BACKED: JVM_OLD` should be equivalent.

Reply:
Oh, that makes sense, thanks :)